swri_geometry_util/src/util.hpp

```diff
@@ -0,0 +1,50 @@
+// *****************************************************************************
```

Comment:
Not sure how these are normally structured, but should this `hpp` file be in `src` or `include`?

Reply:
I agree it also feels weird, but that was ROS2's suggested structure for header files that you don't want exported. That file started with helper functions that didn't create a context to work in, so they were only useful for internal usage.